src/containers/FilmFrontpage/FilmFrontpage.tsx

```diff
@@ -29,13 +28,128 @@ import { GQLFilmFrontPageQuery } from "../../graphqlTypes";
 
-const StyledH1 = styled.h1`
-  ${utils.visuallyHidden}
-`;
+const StyledUl = styled("ul", {
+  base: {
+    display: "grid",
+    gap: "medium",
 
-const StyledMovieList = styled.div`
-  margin: ${spacing.xlarge} 0 ${spacing.xxlarge};
-`;
+    gridTemplateColumns: "1fr 1fr 1fr",
+    tabletDown: {
+      gridTemplateColumns: "1fr 1fr",
+    },
+  },
+});
+
+const StyledSafeLinkButton = styled(SafeLinkButton, {
+  base: {
+    width: "100%",
+    justifyContent: "start",
+    paddingInline: "medium",
+    paddingBlock: "xsmall",
+  },
+});
+
+const StyledNav = styled("nav", {
+  base: {
+    paddingTop: "medium",
+    paddingBottom: "xxlarge",
+  },
+});
+
+const ContentWrapper = styled("div", {
+  base: {
+    width: "100%",
+    marginTop: "medium",
+  },
+});
+
+const getDocumentTitle = (t: TFunction, subject: GQLFilmFrontPageQuery["subject"]) =>
+  htmlTitle(subject?.name, [t("htmlTitles.titleTemplate")]);
+
+const fromNdla = {
+  id: "fromNdla",
+  name: "ndlaFilm.search.categoryFromNdla",
+};
+
+const FilmFrontpage = () => {
+  const allResources = useMemo(
+    () => ({
+      name: "filmfrontpage.resourcetype.all",
+      id: ALL_MOVIES_ID,
+    }),
+    [],
+  );
+
+  const { t, i18n } = useTranslation();
+  const [resourceTypeSelected, setResourceTypeSelected] = useState<MovieResourceType | undefined>(fromNdla);
+  const [loadingPlaceholderHeight, setLoadingPlaceholderHeight] = useState<string>("");
+  const movieListRef = useRef<HTMLDivElement | null>(null);
+
+  const { data: { filmfrontpage, subject } = {}, loading } = useGraphQuery<GQLFilmFrontPageQuery>(filmFrontPageQuery, {
+    variables: { subjectId: "urn:subject:20", transformArgs: { subjectId: "urn:subject:20" } },
+  });
+
+  const about = filmfrontpage?.about?.find((about) => about.language === i18n.language);
+
+  const definedSlideshowMovies = useMemo(
+    () => filmfrontpage?.slideShow.filter((slideshow) => !!slideshow.metaImage),
+    [filmfrontpage?.slideShow],
+  );
+
+  const onChangeResourceType = (resourceType: MovieResourceType) => {
+    const placeholderHeight = `${movieListRef.current?.getBoundingClientRect().height}px`;
+
+    setLoadingPlaceholderHeight(placeholderHeight);
+    setResourceTypeSelected([allResources].concat(movieResourceTypes).find((rt) => rt.id === resourceType.id));
+  };
+
+  const options = useMemo(() => {
+    return [fromNdla, allResources].concat(movieResourceTypes);
+  }, [allResources]);
+
+  return (
+    <>
+      <Helmet>
+        <title>{getDocumentTitle(t, subject)}</title>
+      </Helmet>
+      <SocialMediaMetadata type="website" title={subject?.name ?? ""} description={about?.description} />
+      <main>
+        {loading ? <Spinner /> : definedSlideshowMovies ? <FilmSlideshow slideshow={definedSlideshowMovies} /> : null}
+        <OneColumn wide>
+          <Heading textStyle="heading.medium" id={SKIP_TO_CONTENT_ID}>
+            {t("ndlaFilm.heading")}
+          </Heading>
+          <StyledNav>
```

Comment:
Denne burde referere til headingen under i `aria-labelledby`. Gjerne bruke `useId` for å gjøre det.

Reply:
Skal også være en gap på "4xsmall" mellom heading og kortene.